assets/data-port/import/data/actions.js

```diff
@@ -99,16 +106,17 @@ export const startFetch = () => ( {
 /**
  * Start import.
  *
+ * @param {string}   jobId               The job identifier.
  * @param {Object}   [options]
  * @param {Function} [options.onSuccess] On Success handler.
  * @param {Function} [options.onError]   On Error handler.
  */
-export function* submitStartImport( { onSuccess, onError } = {} ) {
+export function* submitStartImport( jobId, { onSuccess, onError } = {} ) {
 	yield startImport();
 
 	try {
 		const data = yield fetchFromAPI( {
-			path: API_BASE_PATH + 'start',
+			path: API_BASE_PATH + jobId + '/start',
```

Comment:
This url building is used in a few places, for maintainability it could be extracted a helper function.

Reply:
Switched to helper in c86af9cdbe0e98fc6be533abe1774fed744be5e9